fixtures/varytec/event-par-ip65-4in1-14x8w.json

```diff
@@ -0,0 +1,210 @@
+    "Intensity Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Intensity Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Intensity White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Master Intensity": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Shutter / Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "StrobeDuration",
+          "durationStart": "1000ms",
+          "durationEnd": "1ms",
+          "comment": "absolute duration numbers are a guess, there is no documentation on this"
+        },
+        {
+          "dmxRange": [11, 255],
+          "type": "StrobeSpeed",
+          "speedStart": "1Hz",
+          "speedEnd": "15Hz",
+          "comment": "absolute frequency is a guess, there is no documentation on this"
+        }
+      ]
+    },
+    "Effects": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [11, 100],
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "comment": "Color Change without fade (Speed)"
+        },
+        {
+          "dmxRange": [101, 200],
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "comment": "Color Change with fade (Speed)"
+        },
+        {
+          "dmxRange": [201, 255],
+          "type": "SoundSensitivity",
+          "soundSensitivityStart": "low",
+          "soundSensitivityEnd": "high"
+        }
+      ]
+    },
+    "Effects 2": {
+      "name": "Effects",
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [11, 32],
+          "type": "Effect",
+          "effectName": "Colour change with 7 colours"
+        },
+        {
+          "dmxRange": [33, 50],
+          "type": "Effect",
+          "effectName": "Colour change red orange"
+        },
+        {
+          "dmxRange": [51, 68],
+          "type": "Effect",
+          "effectName": "Colour change with 15 colours"
+        },
+        {
+          "dmxRange": [69, 86],
+          "type": "Effect",
+          "effectName": "Colour change with blackout, 4 colours"
+        },
+        {
+          "dmxRange": [87, 100],
+          "type": "Effect",
+          "effectName": "Colour change with 15 colours"
+        },
+        {
+          "dmxRange": [101, 200],
+          "type": "Effect",
+          "effectName": "Colour change with cross-fade, increasing speed"
+        },
+        {
+          "dmxRange": [201, 255],
+          "type": "SoundSensitivity",
+          "soundSensitivityStart": "low",
+          "soundSensitivityEnd": "high"
```

Comment:
I don't see a sensitivity gradient here, but I'm confused by seeing it on channel 9, too, so I can't say it's wrong.